code/lib/cli/src/automigrate/fixes/new-frameworks/utils.ts

```diff
@@ -0,0 +1,88 @@
+import type { Preset, StorybookConfig } from '@storybook/types';
+
+export const packagesMap: Record<string, { webpack5?: string; vite?: string }> = {
+  '@storybook/react': {
+    webpack5: '@storybook/react-webpack5',
+    vite: '@storybook/react-vite',
+  },
+  '@storybook/preact': {
+    webpack5: '@storybook/preact-webpack5',
+    vite: '@storybook/preact-vite',
+  },
+  '@storybook/server': {
+    webpack5: '@storybook/server-webpack5',
+  },
+  '@storybook/ember': {
+    webpack5: '@storybook/ember',
+  },
+  '@storybook/angular': {
+    webpack5: '@storybook/angular',
+  },
+  '@storybook/vue': {
+    webpack5: '@storybook/vue-webpack5',
+    vite: '@storybook/vue-vite',
+  },
+  '@storybook/vue3': {
+    webpack5: '@storybook/vue3-webpack5',
+    vite: '@storybook/vue3-vite',
+  },
+  '@storybook/svelte': {
+    webpack5: '@storybook/svelte-webpack5',
+    vite: '@storybook/svelte-vite',
+  },
+  '@storybook/web-components': {
+    webpack5: '@storybook/web-components-webpack5',
+    vite: '@storybook/web-components-vite',
+  },
+  '@storybook/html': {
+    webpack5: '@storybook/html-webpack5',
+    vite: '@storybook/html-vite',
+  },
+};
```

Comment:
Do we not already have somewhere a map like this? If not, can we put it somewhere where it is more accessible (not in `automigrate`)?

Reply:
This is just useful for the automigration. However, this PR exposes two maps: `rendererPackages` and `frameworkPackages` that list all renderers and all frameworks (that we know of) in `@storybook/core-common`